includes/class-wc-calypso-bridge-admin-setup-wizard.php

```diff
@@ -134,13 +134,172 @@ public function setup_wizard_content() {
+							<?php echo esc_attr( $address_2 ); ?><br>
+						<?php } ?>
+						<?php echo sprintf( '%s, %s %s', esc_attr( $city ), esc_attr( $state ), esc_attr( $postcode ) ); ?><br>
+						<?php echo esc_attr( WC()->countries->countries[ $country ] ); ?>
+					</p>					
+					<button type="button" class="button button-large toggle-store_address_edit" value="<?php esc_attr_e( 'Edit', 'wc-calypso-bridge' ); ?>"><?php esc_html_e( 'Edit', 'wc-calypso-bridge' ); ?></button>
+				</div>
+			</div>
+
+			<div class="store-address-container">
+				<label for="store_address" class="location-prompt"><?php esc_html_e( 'Where is your store based?', 'wc-calypso-bridge' ); ?></label>
+				<input type="text" id="store_address" class="location-input" name="store_address" required value="<?php echo esc_attr( $address ); ?>" />
+				<a href="#" class="toggle-store_address_2"><span class="plus-sign"></span><?php esc_html_e( 'Add address line 2', 'wc-calypso-bridge' ); ?></a>
+				<input type="text" id="store_address_2" class="location-input" name="store_address_2" value="<?php echo esc_attr( $address_2 ); ?>" />
+
+				<div>
+					<label class="location-prompt" for="store_city"><?php esc_html_e( 'City', 'wc-calypso-bridge' ); ?></label>
+					<input type="text" id="store_city" class="location-input" name="store_city" required value="<?php echo esc_attr( $city ); ?>" />
+				</div>
+
+				<div class="store-state-container hidden">
+					<label for="store_state" class="location-prompt">
+						<?php esc_html_e( 'State', 'wc-calypso-bridge' ); ?>
+					</label>
+					<select id="store_state" name="store_state" data-placeholder="<?php esc_attr_e( 'Choose a state&hellip;', 'wc-calypso-bridge' ); ?>" aria-label="<?php esc_attr_e( 'State', 'wc-calypso-bridge' ); ?>" class="location-input wc-enhanced-select dropdown"></select>
+				</div>
+
+				<div class="city-and-postcode">
+					<div>
+						<label class="location-prompt" for="store_country"><?php esc_html_e( 'Country', 'wc-calypso-bridge' ); ?></label>
+						<select id="store_country" name="store_country" required data-placeholder="<?php esc_attr_e( 'Choose a country&hellip;', 'wc-calypso-bridge' ); ?>" aria-label="<?php esc_attr_e( 'Country', 'wc-calypso-bridge' ); ?>" class="location-input wc-enhanced-select dropdown">
+							<?php foreach ( WC()->countries->get_countries() as $code => $label ) : ?>
+								<option <?php selected( $code, $country ); ?> value="<?php echo esc_attr( $code ); ?>"><?php echo esc_html( $label ); ?></option>
+							<?php endforeach; ?>
+						</select>
+					</div>
+					<div>
+						<label class="location-prompt" for="store_postcode"><?php esc_html_e( 'Postcode / ZIP', 'wc-calypso-bridge' ); ?></label>
+						<input type="text" id="store_postcode" class="location-input" name="store_postcode" required value="<?php echo esc_attr( $postcode ); ?>" />
+					</div>
+				</div>
+			</div>
+
+			<div class="store-currency-container">
+			<label class="location-prompt" for="currency_code">
```

Comment:
Some missing tabs/nesting in this div container

Reply:
Fixed!